canonical-web-and-design/ubuntu.com
# Update the server download journey

## Done
- Updated the order to Multipass -> Manual -> MAAS
- Move the CTA button below the option buttons
- Update the content across the three pages
- Added anchors to the manual download page

## QA

- Go to the demo and go to /download/server.
- Check the content matches the [copy doc](https://docs.google.com/document/d/12kr3JvN3ALMdr5IPOheMwOaf-ygvEulFo5D7x-0ag0k/edit)
- Click through the steps checking the content
- Ensure the download works
- Ensure all navigation works

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3608
